xinference/web/ui/src/scenes/launch_model/modelCard.js

```diff
@@ -445,6 +448,26 @@ const ModelCard = ({ url, modelData }) => {
               </Select>
             </FormControl>
           )}
+          <FormControl
+            variant="outlined"
+            margin="normal"
+            size="small"
+            disabled={!modelFormat || !modelSize || !quantization}
+          >
+            <TextField
+              type="number"
+              InputProps={{
+                inputProps: {
+                  min: 0,
+                  max: modelFormat !== 'pytorch' ? 1 : 10000000,
```

Comment:
How about gptq format?

Reply:
If modelFormat == `gptq`, when launching model to 2 cards, an error would be raised:
```
2023-12-06 09:14:41,786 xinference.api.restful_api 501118 ERROR    [address=0.0.0.0:42431, pid=501338] Expected all tensors to be on the same device, but found at least two devices, cuda:1 and cuda:0! (when checking argument for argument index in method wrapper_CUDA__index_select)
Traceback (most recent call last):
  File "/home/lichengjie/workspace/inference/xinference/api/restful_api.py", line 810, in create_chat_completion
    data = await model.chat(prompt, system_prompt, chat_history, kwargs)
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/site-packages/xoscar/backends/context.py", line 227, in send
    return self._process_result_message(result)
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/site-packages/xoscar/backends/context.py", line 102, in _process_result_message
    raise message.as_instanceof_cause()
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/site-packages/xoscar/backends/pool.py", line 657, in send
    result = await self._run_coro(message.message_id, coro)
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/site-packages/xoscar/backends/pool.py", line 368, in _run_coro
    return await coro
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/site-packages/xoscar/api.py", line 306, in __on_receive__
    return await super().__on_receive__(message)  # type: ignore
  File "xoscar/core.pyx", line 558, in __on_receive__
    raise ex
  File "xoscar/core.pyx", line 520, in xoscar.core._BaseActor.__on_receive__
    async with self._lock:
  File "xoscar/core.pyx", line 521, in xoscar.core._BaseActor.__on_receive__
    with debug_async_timeout('actor_lock_timeout',
  File "xoscar/core.pyx", line 526, in xoscar.core._BaseActor.__on_receive__
    result = await result
  File "/home/lichengjie/workspace/inference/xinference/core/utils.py", line 33, in wrapped
    ret = await func(*args, **kwargs)
  File "/home/lichengjie/workspace/inference/xinference/core/model.py", line 67, in wrapped_func
    ret = await fn(self, *args, **kwargs)
  File "/home/lichengjie/workspace/inference/xinference/core/model.py", line 252, in chat
    return await self._call_wrapper(_wrapper)
  File "/home/lichengjie/workspace/inference/xinference/core/model.py", line 200, in _call_wrapper
    return await asyncio.to_thread(_wrapper)
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/asyncio/threads.py", line 25, in to_thread
    return await loop.run_in_executor(None, func_call)
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/lichengjie/workspace/inference/xinference/core/model.py", line 240, in _wrapper
    getattr(self._model, "chat")(prompt, *args, **kwargs)
  File "/home/lichengjie/workspace/inference/xinference/model/llm/pytorch/core.py", line 496, in chat
    c = self.generate(full_prompt, generate_config)
  File "/home/lichengjie/workspace/inference/xinference/model/llm/pytorch/core.py", line 312, in generate
    for completion_chunk, completion_usage in generate_stream(
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 35, in generator_context
    response = gen.send(None)
  File "/home/lichengjie/workspace/inference/xinference/model/llm/pytorch/utils.py", line 157, in generate_stream
    out = model(torch.as_tensor([input_ids], device=device), use_cache=True)
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/lichengjie/.cache/huggingface/modules/transformers_modules/qwen-chat-gptq-7b/modeling_qwen.py", line 1043, in forward
    transformer_outputs = self.transformer(
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/lichengjie/.cache/huggingface/modules/transformers_modules/qwen-chat-gptq-7b/modeling_qwen.py", line 822, in forward
    inputs_embeds = self.wte(input_ids)
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/site-packages/torch/nn/modules/sparse.py", line 162, in forward
    return F.embedding(
  File "/home/lichengjie/miniconda3/envs/xinf/lib/python3.10/site-packages/torch/nn/functional.py", line 2210, in embedding
    return torch.embedding(weight, input, padding_idx, scale_grad_by_freq, sparse)
RuntimeError: [address=0.0.0.0:42431, pid=501338] Expected all tensors to be on the same device, but found at least two devices, cuda:1 and cuda:0! (when checking argument for argument index in method wrapper_CUDA__index_select)
```